requirements/dev-requirements.txt

```diff
@@ -7,4 +7,4 @@ yamllint
 setuptools>=62.0.0
 twine
 wheel
-backports.cached_property
```

Comment:
It's required for lower version of Python

Reply:
ping @aarnphm

Reply:
we already add that under setuptools no need to add it to dev